ci/ansible/kiosk.yml

```diff
@@ -0,0 +1,32 @@
+- hosts: kiosks
+
+  vars:
+    - service_description: 'Herbert kiosk service'
+    - service_user: root
+    - service_file: dist/worker/main.js
+
+  tasks:
+    - name: Install System Dependencies
+      become: yes
+      apt: name={{ item }} update_cache=yes state=latest
+      with_items:
+        - xserver-xorg
+        - xinit
+        - x11-xserver-utils
+        - chromium-browser
+        - matchbox-window-manager
+        - xautomation
+        - unclutter
+
+    - name: Create {{ appDir }}
+      file:
+        path: '{{ appDir }}'
+        state: directory
+
+    - name: Sync Herbert scripts
+      synchronize:
+        src: '{{ srcDir }}/scripts'
+        dest: '{{ appDir }}'
+        recursive: yes
+        delete: yes
+
```

Comment:
I think to install and start the service we need to add

```yml
    - name: Install kiosk service
      become: yes
      template:
        src: ./templates/systemd-service.j2
        dest: /etc/systemd/system/herbert-kiosk.service
        owner: root
        group: root
        mode: '0644'

    - name: Enable and start herbert-kiosk service
      become: yes
      systemd:
        name: herbert-kiosk
        state: restarted
        daemon_reload: yes
        enabled: yes
```

Reply:
I didn't mean to include this. The kiosk starts when the user logs in, via .bashrc calling xinit. Not sure how to add that here. Will move this out of the pull request tho.